core/templates/domain/collection/collection-rights-backend-api.service.ts

```diff
@@ -35,7 +35,8 @@ import { ICollectionRightsBackendDict } from
 export class CollectionRightsBackendApiService {
   // Maps previously loaded collection rights to their IDs.
   collectionRightsCache: Object = {};
-  collectionRightsObjectFactory = new CollectionRightsObjectFactory();
+  collectionRightsObjectFactory: CollectionRightsObjectFactory =
+    new CollectionRightsObjectFactory();
 
   constructor(
     private http: HttpClient,
```

Comment:
Why aren't we using the constructor? E.g.

```suggestion

constructor(
  private http: HttpClient,
  private collectionRightsObjectFactory: CollectionRightsObjectFactory,